docker-compose.yml

```diff
@@ -30,6 +30,9 @@ services:
       READ_MODEL_SPARQL_UPDATE_ENDPOINT: http://fuseki:3030/data-cube-read/update
       STORAGE_FILESYSTEM_PATH: /uploads
       NODE_ENV: production
+      AUTH_JWKS_URI: https://auth.lindas.admin.ch/realms/ref/protocol/openid-connect/certs
+      AUTH_AUDIENCE: account
```

Comment:
@sandhose why does the JWT have this for audience?

Reply:
I think it depends on scopes requested. I should be able to create a new scope that adds a particular audience for this app in keycloak.
The account audience is probably to access keycloak’s account self-management APIs

Reply:
@tpluscode Now it's the `pipelines` audience. Checkout the doc I posted on Zulip about that